### PR TITLE
framework/task_manager : Merge duplicated code about commnunicating w…

### PR DESCRIPTION
**Pull request by jeongchanKim**
…ith task manager

There are many duplicated code.
It can be merged into internal function for avoiding redundancy.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [77e7272576efae7f78829a27a4780ac7341fb138] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [77e7272576efae7f78829a27a4780ac7341fb138] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [2476dbba872066455187b4241e3e4d5343989eb2] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [2476dbba872066455187b4241e3e4d5343989eb2] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [fd9fc350338a5348b78d6ce53b476076ec5bd0c5] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [e6ef0988f04d393f6f781081dc41f717f50fea26] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [e6ef0988f04d393f6f781081dc41f717f50fea26] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [854771bfd49643424db323dbac2e89f69ab41190] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [854771bfd49643424db323dbac2e89f69ab41190] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [3e672f745cc4d8f58fcda8533fb9e3138f894a2f] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [3e672f745cc4d8f58fcda8533fb9e3138f894a2f] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [1147b3b95cdd152ba08f940b9d56b3c40ed948ef] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [1147b3b95cdd152ba08f940b9d56b3c40ed948ef] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [286a9505cb0a372c2e87685653a805524cc28e2b] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [286a9505cb0a372c2e87685653a805524cc28e2b] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [8eba323f3dd675f373b795e6094c3dc258577c31] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [8eba323f3dd675f373b795e6094c3dc258577c31] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [0851954090842371dd8878bda9ebfa1646f98379] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [0851954090842371dd8878bda9ebfa1646f98379] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [f596774a51f5ac7164fe7c84f1bc30dbcbb61450] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [f596774a51f5ac7164fe7c84f1bc30dbcbb61450] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [b6c8ec58afdd4894edb479d56437aa705f1c4b88] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [b6c8ec58afdd4894edb479d56437aa705f1c4b88] - Code Rule Check Result:
framework/src/task_manager/task_manager_core.c:74: ERROR: [IDT_M_TAB] please, no space before tabs

**Comment by seinfra** [comment]
Target : [d4eddd73f77b7e7178b11b5f5c9d42eb87065525] - Code Rule Check Result:
framework/src/task_manager/task_manager_core.c:74: ERROR: [IDT_M_TAB] please, no space before tabs

File: framework/src/task_manager/task_manager_getinfo.c [hunk 1]
```diff
@@ -25,6 +25,29 @@
 #include <task_manager/task_manager.h>
 #include "task_manager_internal.h"
 
+#define TM_GETINFO_ASPRINTF_QUEUE_NAME(request_msg)					\
```
Comment (sunghan-chang) [inline framework/src/task_manager/task_manager_getinfo.c]: How about naming `TM_SET_QUEUE_NAME`?

**Comment by seinfra** [comment]
Target : [d9ca99f21039a8322f7467b361ccf98de733912d] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [d9ca99f21039a8322f7467b361ccf98de733912d] - Code Rule Check Result:
framework/src/task_manager/task_manager_core.c:74: ERROR: [IDT_M_TAB] please, no space before tabs

**Comment by seinfra** [comment]
Target : [3fe35a8057832f640784fcafe42b5a4b815695cb] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [3fe35a8057832f640784fcafe42b5a4b815695cb] - Code Rule Check Result:
framework/src/task_manager/task_manager_core.c:74: ERROR: [IDT_M_TAB] please, no space before tabs

**Pull request merged**